acceptance-tests/src/test/resources/features/UKDrivingLicence/DrivingLicenceCRIAPI.feature

```diff
@@ -1,14 +1,52 @@
-@fraud_CRI_API
-Feature: Fraud CRI API
-
-  @intialJWT_happy_path @pre-merge @dev
-  Scenario: Acquire initial JWT (STUB)
-    Given user has the user identity in the form of a signed JWT string for CRI Id driving-licence-cri-dev
-    When user sends a POST request to session end point
-    Then user gets a session-id
-
-  @intialJWT_happy_path
-  Scenario: Acquire initial JWT (STUB)
-    Given user has the user identity in the form of a signed JWT string for CRI Id driving-licence-cri-build
-    When user sends a POST request to session end point
-    Then user gets a session-id
\ No newline at end of file
+@drivinglicence_CRI_API
+Feature: DrivingLicence CRI API
+
+  @intialJWT_DVLA_happy_path @drivingLicenceCRI_API @pre-merge @dev
```

Comment:
Have we tried running these in build?

Reply:
Have been run against Dev and Shared-dev only at the moment.